Makefile

```diff
@@ -355,3 +355,11 @@ fmt: goimports
 vet:
 	go vet ./...
 
+##################################
+# HELM
+##################################
+
+.PHONY: gen-helm-docs
+gen-helm-docs: ## Generate Helm docs
+	@docker run -v ${PWD}:/work -w /work jnorwood/helm-docs -s file
```

Comment:
It would be good to pin this to a specific version if more than `latest` is tagged.  Whatever is current latest version is fine.

Reply:
Done, pinned to `v1.6.0` (latest).